src/controller.rs

```diff
@@ -44,27 +44,25 @@ pub fn start(
         lang_srv = match language_server_transport::start(&lang.command, &lang.args, &lang.envs) {
             Ok(ls) => ls,
             Err(err) => {
-                // If we think that the server command is not from the default config, then we
-                // send a prominent error to the editor, since it's likely configuration error.
-                let might_be_from_default_config =
-                    !lang.command.contains('/') && !lang.command.contains(' ');
```

Comment:
Makes sense. Running a single lsp command in a buffer without an installed language server shouldn't affect other language servers.
Especially since we already tell the user via the info box.
I wonder if we have more bugs like that.